<ui version="4.0" >
 <class>EditProfileDialog</class>
 <widget class="QWidget" name="EditProfileDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>3</number>
     </property>
     <property name="usesScrollButtons" >
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <widget class="QLabel" name="label_9" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>69</y>
         <width>96</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text" >
        <string>Initial Directory:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>39</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text" >
        <string>Command:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text" >
        <string>Session Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>140</y>
         <width>57</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Icon:</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>200</y>
         <width>590</width>
         <height>126</height>
        </rect>
       </property>
       <property name="title" >
        <string>Default Tab Title</string>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QRadioButton" name="radioButton_4" >
          <property name="text" >
           <string>Program Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_7" >
          <property name="text" >
           <string>Current Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_5" >
          <property name="text" >
           <string>Session Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_6" >
          <property name="text" >
           <string>Custom Format</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="commandEdit" >
       <property name="geometry" >
        <rect>
         <x>111</x>
         <y>39</y>
         <width>450</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="dirSelectButton" >
       <property name="geometry" >
        <rect>
         <x>568</x>
         <y>69</y>
         <width>30</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sessionNameEdit" >
       <property name="geometry" >
        <rect>
         <x>111</x>
         <y>10</y>
         <width>450</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="initialDirEdit" >
       <property name="geometry" >
        <rect>
         <x>111</x>
         <y>70</y>
         <width>450</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="iconSelectButton" >
       <property name="geometry" >
        <rect>
         <x>110</x>
         <y>110</y>
         <width>83</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="iconSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Appearence</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Color Scheme and Background</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <item rowspan="4" row="0" column="0" >
           <widget class="QListView" name="colorSchemeList" >
            <property name="verticalScrollMode" >
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="newColorSchemeButton" >
            <property name="text" >
             <string>New...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QPushButton" name="editColorSchemeButton" >
            <property name="text" >
             <string>Edit...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QPushButton" name="removeColorSchemeButton" >
            <property name="text" >
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Font</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>Preview:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="fontPreviewLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>FONT PREVIEW TEXT</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Text Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Small</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="fontSizeSlider" >
              <property name="minimum" >
               <number>9</number>
              </property>
              <property name="maximum" >
               <number>20</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Large</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editFontButton" >
              <property name="text" >
               <string>Edit Font...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Window Options</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="showMenuBarButton" >
            <property name="text" >
             <string>Show Menu Bar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="alwaysHideTabBarButton" >
            <property name="text" >
             <string>Always hide tab bar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="autoShowTabBarButton" >
            <property name="text" >
             <string>Show tab bar when more than one tab is open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="alwaysShowTabBarButton" >
            <property name="text" >
             <string>Always show tab bar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Keyboard Setup</string>
      </attribute>
      <layout class="QGridLayout" >
       <item rowspan="4" row="0" column="0" >
        <widget class="QListView" name="keyBindingList" />
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="newKeyBindingsButton" >
         <property name="text" >
          <string>New...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="editKeyBindingsButton" >
         <property name="text" >
          <string>Edit...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="removeKeyBindingsButton" >
         <property name="text" >
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5" >
      <attribute name="title" >
       <string>Scrolling</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="title" >
          <string>Scrolling</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="disableScrollbackButton" >
            <property name="text" >
             <string>Disable scrollback</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QRadioButton" name="fixedScrollbackButton" >
              <property name="text" >
               <string>Fixed number of lines: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_4" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="unlimitedScrollbackButton" >
            <property name="text" >
             <string>Unlimited scrollback</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7" >
         <property name="title" >
          <string>Scroll Bar </string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="scrollBarLeftButton" >
            <property name="text" >
             <string>Show on left side</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="scrollBarRightButton" >
            <property name="text" >
             <string>Show on right side</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="scrollBarHiddenButton" >
            <property name="text" >
             <string>Hidden</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Terminal Features:</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="checkBox_2" >
            <property name="text" >
             <string>Allow blinking text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3" >
            <property name="text" >
             <string>Enable flow control using Ctrl+S , Ctrl+Q</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_5" >
            <property name="text" >
             <string>Allow programs to resize terminal window</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
